opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -328,33 +328,46 @@ class SimulatorFullyImplicitBlackoilEbos
             std::vector<std::vector<double>> currentFluidInPlace;
             currentFluidInPlace = solver->computeFluidInPlace(fipnum);
             std::vector<double> currentFluidInPlaceTotals = FIPTotals(currentFluidInPlace, state);
+			      const std::string version = moduleVersionName();
```

Comment:
Use spaces instead of tabs. Ensure whitespace and indentation is consistent with surrounding code.